spharpy/classes/audio.py

```diff
@@ -0,0 +1,191 @@
+from pyfar import Signal
+from spharpy.spherical import n3d_to_maxn, n3d_to_sn3d_norm
+from spharpy.spherical import fuma_to_nm, nm_to_fuma, acn_to_nm, nm_to_acn
+from spharpy.spherical import renormalize, change_channel_convention
+import numpy as np
+
+
+class SphericalHarmonicSignal(Signal):
+    """Create audio object with spherical harmonics coefficients in time or
+    frequency domain.
+
+    Objects of this class contain spherical harmonics coefficients which are
+    directly convertible between time and frequency domain (equally spaced
+    samples and frequency bins), the channel conventions `acn` and `fuma`, as
+    well as the normalizations `n3d`, `sn3d`, or `maxn`. The default
+    parameters for `basis_type`, `normalization`, and `channel_convention`
```

Comment:
```suggestion
    well as the normalizations ``'n3d'``, ``'sn3d'``, or ``'maxn'``. The default
    parameters for ``basis_type``, ``normalization``, and ``channel_convention``
```

Reply:
I would rather not treat this as code but simply as abbreviations
```suggestion
    samples and frequency bins), the channel conventions ACN and FUMA, as
    well as the normalizations N3D, SN3D, or MaxN. The default
    parameters for `basis_type`, `normalization`, and `channel_convention`
```

Reply:
agree